bundles/org.smarthomej.binding.tr064/README.md

```diff
@@ -132,6 +132,7 @@ The call-types are the same as provided by the FritzBox, i.e. `1` (inbound), `2`
 | `wifi5GHzEnable`           | `Switch`                  |          | Enable/Disable the 5.0 GHz WiFi device.                        |
 | `wifiGuestEnable`          | `Switch`                  |          | Enable/Disable the guest WiFi.                                 |
 | `macOnline`                | `Switch`                  |     x    | Online status of the device with the given MAC                 |
+| `macOnlineIpAddress`       | `String`                  |     x    | IP of the MAC (uses same parameter as `macOnline`)             |
```

Comment:
The documentation tells us that the Channel is an advanced Channel. But it is created a non advanced.

Same for `macOnline`.

Reply:
Done